tap/passive_tapper.go

```diff
@@ -94,6 +95,8 @@ var ownIps []string             // global
 var hostMode bool               // global
 var extensions []*api.Extension // global
 
+const baseStreamChannelTimeoutMs int = 5000 * 100
```

Comment:
Use `time.Millisecond` or `time.Nanosecond`.

Reply:
@nimrod-up9 if we make it `time.Duration` then it has to be converted back to `int/float` in here:

https://github.com/up9inc/mizu/blob/6a6575a72793f102311116edd237c797515420f2/tap/passive_tapper.go#L224

While the value is converted to `time.Duration` eventually in here:

https://github.com/up9inc/mizu/blob/6a6575a72793f102311116edd237c797515420f2/tap/passive_tapper.go#L226

So converting it to `time.Duration` will probably increase the amount CPU instructions needed to run.